server.js

```diff
@@ -73,7 +82,7 @@ var connectToClusterStateServer = function () {
   var stateSocket = scClient.connect(scStateSocketOptions);
 
   stateSocket.on('error', (err) => {
-    console.error(err);
+    (LOG_LEVEL > 0) && console.error(err);
```

Comment:
Optional (depends on what you decide in scc-state PR), but maybe consider using `if` statement here? Consistency is good.
 The rest of SC uses `if` for doing comparison checks. See https://github.com/SocketCluster/socketcluster/blob/master/index.js#L568